coldfront/core/project/templates/project/project_removal/project_remove_users.html

```diff
@@ -91,65 +138,18 @@ <h1>Remove users from project: {{project.title}}</h1>
-                                                                    <span aria-hidden="true">&times;</span>
-                                                                </button>
-                                                            </div>
-
-                                                            <div class="modal-body">
-                                                                <label>Are you sure? This user will immediately be disallowed from submitting jobs under this project.</label>
-
-                                                                <br><b>User:</b> {{ user.username }}
-                                                                <br><b>Project:</b> {{ project.title }}
-                                                                <br><b>Role:</b> {{ user.role }}
-                                                                <br><b>Status:</b> {{ user.status }}
-
-                                                                <input type="hidden" value="{{ user.username }}" name="username">
-                                                            </div>
-
-                                                            <div class="modal-footer">
-                                                                <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                                                                <button class="btn btn-danger" type="submit">
-                                                                    <i class="fas fa-user-plus" aria-hidden="true"></i>
-                                                                    Submit
-                                                                </button>
-                                                            </div>
-                                                        </div>
-                                                    </form>
-                                                </div>
-                                            </div>
-
-                                            <button class="btn btn-danger" data-toggle="modal" data-target="#remove{{ user.username }}Modal">
-                                                <i class="fas fa-user-plus" aria-hidden="true"></i>
-                                                Remove
-                                            </button>
+                                        <td class="d-flex justify-content-around">
+                                                <input type="checkbox"
+                                                       class="form-check-input"
+                                                       name="users_selected_for_removal"
+                                                       value="{{ user.username }},{{user.email}},{{user.role}},{{user.status}}"
+                                                       id="user{{ user.username }}"
```

Comment:
We ran into a bug on MyLRC wherein a username contained a ".". The removal modal for that user, which had an id of `"remove{{ user.username }}Modal"` (e.g., "removeusername.withdotModal"), couldn't be referenced by `id` because of it. And therefore, the modal wouldn't open when the button to open it was pressed.

That specific issue won't be relevant anymore, given this PR, but it's probably better to use the user's PK/ID instead in case this `input` needs to be referenced by `id`.